ROADMAP.md

```diff
@@ -0,0 +1,51 @@
+# ROADMAP
+
+Truss is a living, breathing project. Our goal is to make it as easy as possible to get value from your work by providing a simple, yet powerful way to package and deploy machine learning models into production systems. We're also building a community of developers who are passionate about building machine learning systems. Our roadmap is a way to keep you informed about the progress of our project, and what we're working on.
```

Comment:
tip: I've found the rewrap vscode plugin very useful in wrapping markdown text. https://marketplace.visualstudio.com/items?itemName=stkb.rewrap